content/en/docs/gke/private-clusters.md

```diff
@@ -1,301 +1,235 @@
-
-* [Documentation on how to use Kubeflow with private GKE and VPC service controls](https://github.com/kubeflow/website/issues/1705)
-* [Replicating Docker images to private Container Registry](https://github.com/kubeflow/kubeflow/issues/3210)
-* [Installing Istio for Kubeflow on private GKE](https://github.com/kubeflow/kubeflow/issues/3650)
-* [Profile-controller crashes on GKE private cluster](https://github.com/kubeflow/kubeflow/issues/4661)
-* [kfctl should work with private GKE without public endpoint](https://github.com/kubeflow/kfctl/issues/158)
-{{% /alert %}}
-
-This guide describes how to secure Kubeflow using [VPC Service Controls](https://cloud.google.com/vpc-service-controls/docs/) and private GKE.
-
-Together these two features signficantly increase security
-and mitigate the risk of data exfiltration.
-
-  * VPC Service Controls allow you to define a perimeter around
-    Google Cloud Platform (GCP) services.
-    
-    Kubeflow uses VPC Service Controls to prevent applications
-    running on GKE from writing data to GCP resources outside
-    the perimeter.
 
-  * Private GKE removes public IP addresses from GKE nodes making
-    them inaccessible from the public internet.
+These instructions explain how to deploy Kubeflow using private GKE.
 
-    Kubeflow uses IAP to make Kubeflow web apps accessible
-    from your browser.
+1. Follow the blueprint instructions to setup a management cluster
 
-VPC Service Controls allow you to restrict which Google services are accessible from your
-GKE/Kubeflow clusters. This is an important part of security and in particular
-mitigating the risks of data exfiltration.
+1. As a work around for (kubeflow/gcp-blueprints#32)[https://github.com/kubeflow/gcp-blueprints/issues/32]
+   modify the containercluster CRD schema in your management cluster to include missing fields
 
-For more information refer to the [VPC Service Control Docs](https://cloud.google.com/vpc-service-controls/docs/overview).
+   * See directions in that issue.
 
-Creating a [private Kubernetes Engine cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters)
-means the Kubernetes Engine nodes won't have public IP addresses. This can improve security by blocking unwanted outbound/inbound
-access to nodes. Removing IP addresses means external services (such as GitHub, PyPi, and DockerHub) won't be accessible
-from the nodes. Google services (such as BigQuery and Cloud Storage) are still accessible.
+1. Fetch the blueprint
 
-Importantly this means you can continue to use your [Google Container Registry (GCR)](https://cloud.google.com/container-registry/docs/) to host your Docker images. Other Docker registries (for example, DockerHub) will not be accessible. If you need to use Docker images
-hosted outside GCR you can use the scripts provided by Kubeflow to mirror them to your GCR registry.
+   ```
+   kpt pkg get https://github.com/kubeflow/gcp-blueprints.git/kubeflow@master ./${PKGDIR}
+   ```
```

Comment:
Suggestion: 

- Add: "where `{PKGDIR}` is the directory (for Kubeflow that you created earlier)?

Reply:
Added text this directory gets created.